CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
-# Changelog
+# Change Log
```

Comment:
This file should be reverted so semantic-release can work.

Reply:
Nice! I don't know well how `semantic-release` works, so I ended up modifying this file. Reverted!